src/modules/todo/hooks/useFetchTodos.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query'
+import { TodoApiService } from "../api"
+
+export function useFetchTodos() {
+    const { isLoading, data } = useQuery({
```

Comment:
I thought about this and decided to return only what I know will work,
since useQuery might need additional data for other features to work

But I guess this makes sense that after I learn more I won't have to update API

Reply:
The thing is you will always have the same functionally retruned from useQuery, you can read about it here https://tanstack.com/query/v5/docs/react/reference/useQuery